airbyte-integrations/connectors/source-github/source_github/streams.py

```diff
@@ -276,7 +276,9 @@ def read_records(
 class IncrementalGithubStream(SemiIncrementalGithubStream):
     def request_params(self, stream_state: Mapping[str, Any], stream_slice: Mapping[str, Any] = None, **kwargs) -> MutableMapping[str, Any]:
         params = super().request_params(stream_state=stream_state, **kwargs)
-        params["since"] = self.get_starting_point(stream_state=stream_state, repository=stream_slice["repository"])
+        since_params = self.get_starting_point(stream_state=stream_state, repository=stream_slice["repository"])
```

Comment:
`self._start_date` could be None?

Reply:
Yep, in some cases, when it is necessary to read the Incremental stream as Full Refresh, while not having a `start_date` and passing an empty string there, as in [this](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connectors/source-github/source_github/streams.py#L321) case, for example.